components/blueprints/Lists.vue

```diff
@@ -94,7 +95,7 @@ watch([currentPage, itemsPerPage, activeTag, searchQuery], ([pageVal, itemVal, t
     }
     timer = setTimeout(async () => {
 
-        const { data } = await useFetch(`https://api.kestra.io/v1/blueprints?page=${(itemVal != oldItemVal) || (tagVal != oldTagVal) ? 1 : pageVal}&size=${itemVal}${Object.keys(tagVal).length && tagVal.name != 'All tags' ? `&tags=${tagVal.id}` : ''}${searchVal.length ? `&q=${searchVal}` : ''}`)
+        const { data } = await useFetch(`${config.public.apiUrl}/blueprints?page=${(itemVal != oldItemVal) || (tagVal != oldTagVal) ? 1 : pageVal}&size=${itemVal}${Object.keys(tagVal).length && tagVal.name != 'All tags' ? `&tags=${tagVal.id}` : ''}${searchVal.length ? `&q=${searchVal}` : ''}`)
```

Comment:
You could use `querystring` module to build the whole query.

Reply:
I think it would be noise on this PR, maybe you can create an issue to unify this on every api calls ?

Reply:
The querystring would make it a bit easier to destructure.

```js
import querystring from querystring

const qs = querystring.stringify({
  page: currentPage.value,
  size: itemsPerPage.value,
  tags: route.query.tags ? activeTag.value.id : undefined,
  q: route.query.q ? searchQuery.value : undefined,
});

// Output: "page=1&size=10&tags=selectedTagID&q=searchQueryValue"

```